_components/button-group/guidance/variants/default/accessibility.md

```diff
@@ -1 +1,4 @@
+{% include accessibility-tests/a11y-note.html %}
+ 
+{:.usa-content-list}
```

Comment:
**suggestion:** Can you add this to the segmented button group section as well? (in file `_components/button-group/guidance/variants/segmented/accessibility.md`)? 

Open to discussion if it feels weird to have this alert in two places on the page.

Reply:
Updated in [84de099](https://github.com/uswds/uswds-site/pull/3009/commits/84de0992533865fadd59f9a8c415d38a155d7a71)